### PR TITLE
GLT-1255: Make Date of receipt to default to today's date when creating samples…

### PR DESCRIPTION
**Pull request by ndrein-oicr**
… and receiving libraries

Changes bulk editing samples/libraries, but only when creating samples/libraries.

### CODE REVIEW

**Comment by sqwci** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [hot_sample.js#L597](https://github.com/TGAC/miso-lims/blob/5ae22d43c0683f35ddc438cacc33b2cce8360bbb/miso-web/src/main/webapp/scripts/hot_sample.js#L597): Remove this usage of alert(...). [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)
1. ![MINOR][MINOR] [hot_sample.js#L602](https://github.com/TGAC/miso-lims/blob/5ae22d43c0683f35ddc438cacc33b2cce8360bbb/miso-web/src/main/webapp/scripts/hot_sample.js#L602): Remove this usage of alert(...). [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: miso-web/src/main/webapp/scripts/lims.js [hunk 1]
```diff
@@ -467,7 +467,18 @@ var Utils = Utils
...
         }
         return val || null;
-      }
+      },
+
+      // Return current date in format YYYY-MM-DD
+      getCurrentDate: function() {
+        var today = new Date();
+        var dd = today.getDate();
+        var mm = today.getMonth();
```
Comment (callunity) [inline miso-web/src/main/webapp/scripts/lims.js]: Month is (regrettably) zero-indexed.

Comment (ndrein-oicr) [reply miso-web/src/main/webapp/scripts/lims.js]: :scream: 

Will fix.

**Comment by sqwci** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [hot_sample.js#L597](https://github.com/TGAC/miso-lims/blob/8831705bdaa654cac239429a4973689732278d79/miso-web/src/main/webapp/scripts/hot_sample.js#L597): Remove this usage of alert(...). [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)
1. ![MINOR][MINOR] [hot_sample.js#L602](https://github.com/TGAC/miso-lims/blob/8831705bdaa654cac239429a4973689732278d79/miso-web/src/main/webapp/scripts/hot_sample.js#L602): Remove this usage of alert(...). [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)
1. ![MINOR][MINOR] [lims.js#L326](https://github.com/TGAC/miso-lims/blob/8831705bdaa654cac239429a4973689732278d79/miso-web/src/main/webapp/scripts/lims.js#L326): Remove this usage of alert(...). [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)
1. ![MINOR][MINOR] [lims.js#L551](https://github.com/TGAC/miso-lims/blob/8831705bdaa654cac239429a4973689732278d79/miso-web/src/main/webapp/scripts/lims.js#L551): Remove this usage of confirm(...). [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)
1. ![MINOR][MINOR] [lims.js#L649](https://github.com/TGAC/miso-lims/blob/8831705bdaa654cac239429a4973689732278d79/miso-web/src/main/webapp/scripts/lims.js#L649): Remove this usage of confirm(...). [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by sqwci** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [hot_sample.js#L597](https://github.com/TGAC/miso-lims/blob/c257ad6ccaccd9c5a70b0ff5eed96f3d863473b7/miso-web/src/main/webapp/scripts/hot_sample.js#L597): Remove this usage of alert(...). [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)
1. ![MINOR][MINOR] [hot_sample.js#L602](https://github.com/TGAC/miso-lims/blob/c257ad6ccaccd9c5a70b0ff5eed96f3d863473b7/miso-web/src/main/webapp/scripts/hot_sample.js#L602): Remove this usage of alert(...). [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)
1. ![MINOR][MINOR] [lims.js#L326](https://github.com/TGAC/miso-lims/blob/c257ad6ccaccd9c5a70b0ff5eed96f3d863473b7/miso-web/src/main/webapp/scripts/lims.js#L326): Remove this usage of alert(...). [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)
1. ![MINOR][MINOR] [lims.js#L551](https://github.com/TGAC/miso-lims/blob/c257ad6ccaccd9c5a70b0ff5eed96f3d863473b7/miso-web/src/main/webapp/scripts/lims.js#L551): Remove this usage of confirm(...). [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)
1. ![MINOR][MINOR] [lims.js#L649](https://github.com/TGAC/miso-lims/blob/c257ad6ccaccd9c5a70b0ff5eed96f3d863473b7/miso-web/src/main/webapp/scripts/lims.js#L649): Remove this usage of confirm(...). [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**